apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/GoogleLoggerProvider.cs

```diff
@@ -50,6 +50,12 @@ internal GoogleLoggerProvider(IConsumer<LogEntry> consumer, LogTarget logTarget,
             _logTarget = GaxPreconditions.CheckNotNull(logTarget, nameof(logTarget));
             _loggerOptions = GaxPreconditions.CheckNotNull(loggerOptions, nameof(loggerOptions));
             _serviceProvider = serviceProvider;
+
+            var writer = loggerOptions.LoggerDiagnosticsOutput;
+            if (writer != null)
+            {
+                ((GoogleLogger) CreateLogger("SampleLogName")).WriteDiagnostics(writer);
```

Comment:
Is "SampleLogName" correct here?

Reply:
Yes. It's not currently used in the method anyway, but I figured it would be a reasonable value to end up being written to diagnostics if we changed things so it *was* used.
(This isn't the log name in the monitored resource - it's the ASP.NET Core log name. Very confusing, I know.)

Would a comment of basically the above be useful?

Reply:
Yes, please do add a comment - it's very non-obvious!

Reply:
Done. PTAL.